x/ccv/provider/keeper/validator_set_update.go

```diff
@@ -101,12 +101,12 @@ func DiffValidators(
 ) []abci.ValidatorUpdate {
 	var updates []abci.ValidatorUpdate
 
-	isCurrentValidator := make(map[string]types.ConsumerValidator)
+	isCurrentValidator := make(map[string]types.ConsumerValidator, len(currentValidators))
```

Comment:
Passing the size of the map here and below based on the "Code Efficiency Improvements and Optimization Opportunities."